frontend/assets/stylesheets/_vars.scss

```diff
@@ -160,7 +159,33 @@ $c-tone-masterclass-accent: #fcdd03;
 $c-tone-friend: $c-neutral1;
 $c-tone-partner: map-get($pasteup-palette, news-main-2);
 $c-tone-patron: map-get($pasteup-palette, news-main-1);
-$c-tone-tint: $c-neutral7;
+
+// Usage: tone('live', 'accent')
+//        tone('live') => defaults to 'base' key
+$palette-tones: (
```

Comment:
This is the one bit I'm not 100% convinced by. Given the number of tone colours we now have it felt nice to try organising them using a map, but as we are not using maps for all colours it might be better to stick with convention for now.